castle/cms/static/components/manage-archives.js

```diff
@@ -300,12 +330,29 @@ require([
       ]);
     },
 
+    renderNav: function(){
+      var that = this;
+      return D.div({}, [
+        D.button({
+          className: 'plone-btn plone-btn-default',
+          disabled: that.state.page <= 1,
+          onClick: that.navigatePrevClicked
+        }, 'Previous Page'),
+        D.button({
+          className: 'plone-btn plone-btn-default pull-right',
+          disabled: that.state.items.length < 100,
```

Comment:
Should the 100 here be the per_page value which is 1000 in the python?  e.g. if there were 200 items the Next button will be enabled but no more pages available on backend?

Reply:
oh yeah, thanks :) TODO: never hard code things